workitem/workitemtype.go

```diff
@@ -27,32 +28,38 @@ const (
 	SystemIteration           = "system.iteration"
 	SystemArea                = "system.area"
 
-	// base item type with common fields for planner item types like userstory, experience, bug, feature, etc.
-	SystemPlannerItem = "planneritem"
-
-	SystemUserStory        = "userstory"
-	SystemValueProposition = "valueproposition"
-	SystemFundamental      = "fundamental"
-	SystemExperience       = "experience"
-	SystemFeature          = "feature"
-	SystemScenario         = "scenario"
-	SystemBug              = "bug"
-
 	SystemStateOpen       = "open"
 	SystemStateNew        = "new"
 	SystemStateInProgress = "in progress"
 	SystemStateResolved   = "resolved"
 	SystemStateClosed     = "closed"
 )
 
+// Never ever change these UUIDs!!!
```

Comment:
These are just there for test/historical purpose right?  As each of these would get a new unique ID when 'applied' to a space via the space template.